sdk/communication/Azure.Communication.Calling.Server/src/swagger/Sample-Swagger.json

```diff
@@ -1064,8 +1030,8 @@
                     "description": "The value to identify context of the operation.",
                     "type": "string"
                 },
-                "resourceId": {
-                    "description": "The media resource id of the play audio request. A ResourceId value\r\nindicates the media resource should be cached within the runtime service\r\nwhich handles playing audios.",
+                "audioFileId": {
```

Comment:
Hey @navali-msft, the renaming of the AudioFileId from ResourceId was not updated in the SDK and it seems that it got pulled into this PR, I have updated and fixed the test regarding this change. Please review this and let me know if its okay to keep in this PR?

Reply:
Discussed offline with @navali-msft and confirmed this, Resolving the comment